dbms/src/Interpreters/Context.cpp

```diff
@@ -1697,11 +1698,22 @@ std::shared_ptr<TraceLog> Context::getTraceLog()
     auto lock = getLock();
 
     if (!shared->system_logs || !shared->system_logs->trace_log)
-        return nullptr;
+        return {};
 
     return shared->system_logs->trace_log;
 }
 
+std::shared_ptr<TextLog> Context::getTextLog()
+{
+    auto lock = getLock();
+
+    if (!shared->system_logs)
```

Comment:
?

Reply:
It is written for SYSTEM FLUSH LOGS functionality. Because it requires all system logs as Context’s field.

Reply:
Null pointer dereference.

Reply:
ohh, yes..